sbnanaobj/StandardRecord/SRTrkLikelihoodPID.cxx

```diff
@@ -0,0 +1,34 @@
+////////////////////////////////////////////////////////////////////////
+// \file    SRTrkLikelihoodPID.cxx
+// \brief   An SRTrkLikelihoodPID is a high level track ParticlePID object for
+//          for larana LikelihoodPIDAlg results.
+////////////////////////////////////////////////////////////////////////
+
+#include "sbnanaobj/StandardRecord/SRTrkLikelihoodPID.h"
+
+#include <limits>
+
+namespace caf
+{
+
+  SRTrkLikelihoodPID::SRTrkLikelihoodPID():
+    pdg(-999),
+    pid_ndof(-99),
+    lambda_muon(std::numeric_limits<float>::signaling_NaN()),
+    lambda_pion(std::numeric_limits<float>::signaling_NaN()),
+    lambda_proton(std::numeric_limits<float>::signaling_NaN())
+  {  }
+
+  SRTrkLikelihoodPID::~SRTrkLikelihoodPID(){  }
+
+  void SRTrkLikelihoodPID::setDefault()
+  {
+    pdg           = -5;
```

Comment:
I would not leave [anti-bottom quark](https://pdg.lbl.gov/2025/web/viewer.html?file=../reviews/rpp2024-rev-monte-carlo-numbering.pdf) as default value, just in case. I'd stick with an invalid value like `0`.
```suggestion
    pdg           = 0;
```